sw/device/silicon_creator/rom_ext/rom_ext_manifest.h

```diff
@@ -0,0 +1,25 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_ROM_EXT_MANIFEST_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_ROM_EXT_MANIFEST_H_
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/manifest.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Return the pointer to rom_ext manifest on the active flash bank.
```

Comment:
maybe you could drop a comment here why this function is different than https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/manifest_def.c;l=39?q=manifest_def_get&ss=opentitan%2Fopentitan  (therefore, why is is needed in addition to the current solution for getting the manifest of the currently executing slot)?